en/telemetry/sik_radio.md

```diff
@@ -0,0 +1,32 @@
+# SiK Radio
+
+[SiK radio](https://github.com/LorenzMeier/SiK) is a collection of firmware and tools for telemetry radios.
+
+Hardware for the SiK radio can be obtained from various manufacturers/stores in variants that support different range and form factors. Typically you will need a pair of devices - one for the vehicle and one for the ground station. 
+
+## Purchase {#vendors}
+
+* 3DR Radio \(small\) - [US (915MHz)](https://store.3dr.com/products/915-mhz-telemetry-radio) / [European (433MHz)](https://store.3dr.com/products/433-mhz-telemetry-radio)
+* [HK Radio](http://www.hobbyking.com/hobbyking/store/uh_viewitem.asp?idproduct=55559) \(small\)
+* [RFD900u](http://rfdesign.com.au/products/rfd900u-modem/) \(small\)
+* [RFD900](http://rfdesign.com.au/products/rfd900-modem/) \(long range\)
+
+![SiK Radio](../../assets/hardware/telemetry/sik_radio.jpg)
+
+## Setup/Configuration
+
+The ground station-based radio is connected via USB (essentially plug-n-play). 
+
+The vehicle is connected to the flight-controller's `TELEM1` port, and typically requires no further configuration.
+
+
+## Firmware Update
```

Comment:
@bkueng I've updated this section in line with your comments. Can you please check it?

I tried this with 3DR radio, but it is not detected by QGC for firmware update. Can you confirm that if this *did* work the latest version, including support for MAVLink2, would be installed?

Reply:
Yes, I can confirm that at least it used to work, because I've done that.